README.md

```diff
@@ -1,36 +1,36 @@
-golem-scala
-===========
+sbt-wasm-component
+==================
 
 Avoid any boilerplate in your project by using just one annotation to export your Golem worker from Scala to JS.
 
 Setup
 -----
 
-Add golem-scala as a dependency in `project/plugins.sbt`:
+Add sbt-wasm-component as a dependency in `project/plugins.sbt`:
 
 ```scala
-addSbtPlugin("cloud.golem" % "golem-scala" % "x.y.z")
+addSbtPlugin("cloud.golem" % "sbt-wasm-component" % "x.y.z")
 ```
 
 Usage
 -----
 
-Golem-scala is automatically loaded, it just needs to be enabled with `enablePlugins(GolemScalaPlugin)` in your `build.sbt`:
+The WASM component plugin is automatically loaded, it just needs to be enabled with `enablePlugins(WasmComponentPlugin)` in your `build.sbt`:
 
 ```scala
 ThisBuild / version := "0.1.0-SNAPSHOT"
 ThisBuild / scalaVersion := "2.13.13"
 
 lazy val root = (project in file("."))
-  .enablePlugins(GolemScalaPlugin)
+  .enablePlugins(WasmComponentPlugin)
```

Comment:
please add the configuration of the wit file name

Reply:
Yes, in a new PR :D